### ISSUE TITLE
Split lines does not work if options is not passed

### ISSUE THREAD

**Issue by jmigual** [evanplaice/jquery-csv#78]
If you call:
```javascript
$.csv.parsers.splitLines(csv)
```
It does not work and gives the following message:
```
Uncaught TypeError: Cannot read property 'separator' of undefined
```

**Comment by callumstrubi** [comment]
I can confirm this too

**Comment by fgerschau** [comment]
Would a proper solution be using the default values of `$.csv.defaults`?
Also, I notices that there's the same problem with all function inside $.csv.parsers. If you don't mind I would like to start contributing with a solution @evanplaice

**Comment by evanplaice** [comment]
@fgerschau Oh, I just realized why this bug exists. splitLines and parsers are undocumented and don't support optional args because I only ever intended to use them internally. They should only be called by other public API functions that will guarantee to always provide the options.

I'm honestly surprised somebody went to the effort to dig through the code to find them to begin with. If exposing them as part of the public API makes sense for some users, I'm all for it.

You're definitely going in the right direction. PRs are always welcome if you'd like to make this happen.

If these functions are going to be exposed publicly, I'd prefer that two things are added:

1. a test or tests that prove the functionality
2. a mention of there usage somewhere in the documentation

If you'd like to make this happen, I'll try my best to help.

**Comment by fgerschau** [comment]
@evanplaice ok, cool. I'll start working on that 👍

**Comment by fgerschau** [comment]
Opened a PR here [#99](https://github.com/evanplaice/jquery-csv/pull/99)

**Comment by evanplaice** [comment]
Fixed with https://github.com/evanplaice/jquery-csv/pull/99.

Included in release 0.8.8
